README.md

```diff
@@ -342,8 +342,8 @@ cloud. For more information about HDR capture, visit our
 
 The capture settings available in the `zivid_camera` node matches the settings in the Zivid API.
 To become more familiar with the different settings and what they do, see the API reference for the
-[Settings](http://www.zivid.com/hubfs/softwarefiles/releases/2.0.0+7c4918cf-14/doc/cpp/classZivid_1_1Settings.html)
-and [Settings2D](http://www.zivid.com/hubfs/softwarefiles/releases/2.0.0+7c4918cf-14/doc/cpp/classZivid_1_1Settings2D.html)
+[Settings](http://www.zivid.com/hubfs/softwarefiles/releases/2.1.0+d2007e12-1/doc/cpp/classZivid_1_1Settings.html)
```

Comment:
For DRY, it would be nice to make a variable for the version string.
Not sure if that's possible in Markdown. Do you know, @apartridge?

Reply:
Unfortunately this is not possible

Reply:
Ah, unfortunate, that would be nice.
Then I'll just resolve this comment.